### DISCUSSION TITLE
invalid configuration in DNS settings

### DISCUSSION THREAD

**Discussion by raymondtju** [vercel/community#4124] [Help]
### Summary

i have domain registered in godaddy, has add DNS A Record into it,
but in vercel, it says Invalid Configuration

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/75233529/e4939a13-9a17-4155-9f73-1e12055b79ab)

**Comment by swarnava** [comment]
Hi there,

It looks like the DNS hasn't been propagated yet: https://dnschecker.org/#A/zez.pw. Once the `A` record is fully, your domain should be up and running.

**Reply by raymondtju** [reply]
I have already contact the GoDaddy CS but they said that vercel don't read well the A record?

DNS was well set about 1 week ago

**Reply by swarnava** [reply]
Unfortunately it's not Vercel specific issue. You can run `dig` in your local system or use third party system line [DNSChecker](https://dnschecker.org/#A/zez.pw) where you can clearly see that the DNS hasn't propagated yet.
